### DISCUSSION TITLE
Unable to deploy SumatraPDF 3.4.6 through MDT

### DISCUSSION THREAD

**Discussion by M1k3l0ny** [sumatrapdfreader/sumatrapdf#3537] [Q&A] [answered]
Hey,

I'm trying to deploy SumatraPDF software, among others, as an application within Microsoft Deployment Toolkit (MDT) as a step of a task sequence (Install applications), but seems impossible to install. But if I run a CMD with the same command-line arguments, it works like charm. That's weird...

This is the command-line parameters I'm using to deploy the software via MDT:

**SumatraPDF-3.4.6-64-install.exe -s -d "C:\Program Files\SumatraPDF" -all-users -log**

As I said if I run the same command in a machine through CMD, everything works fine. Don't know why, but when it comes to deploy it via MDT the progress bar sticks indefinitely and nothing happens (neither log pops up).

I've been able to install other programs (EXE and MSI) succesfully, so seems something is going wrong with this specific software.

Thanks in advance!

Best regards

**Comment by GitHubRulesOK** [comment]
Hmmm the -all-users option was a later addition and may alter the UAC state so without silent the install will hang there waiting for elevation. but run from elevated prompt will naturally run silent and deep. (Darn now I need to uninstall to retest)

I suspect the state of needing elevation is hidden by running silent when using MDT

**Reply by M1k3l0ny** [reply]
Hey,

I've tried a lot of combinations, even just using the -s parameter, and it's not installed...

Thanks,

Best regards

**Comment by M1k3l0ny** [comment, marked as answer]
Good morning,

After several attemps of deploying SumatraPDF through MDT, I finally gave up and tried another way.

I finally created a small Powershell script with the parameters of installation, and a new step wtihin task sequence (Run Command Line) invoking Powershell to execute it once you arrive at Desktop in Windows. Worked like a charm.

I leave the script here:

$Arguments = @(
        “-s”
        “-all-users”
        )
Start-Process -FilePath “PATHWHEREEXEISLOCATED” -ArgumentList $Arguments -Wait -NoNewWindow

Parameters of the TS (Run Command Line):

Command line: Powershell.exe -ExecutionPolicy Bypass -File "%ScriptRoot%\SCRIPT.ps1"
Start in: %ScriptRoot%
